### PR TITLE
docs: add details for instructions in the onboarding docs

### PR DESCRIPTION
**Pull request by wangyyt1013**
Signed-off-by: Yuanyuting Wang <yuanyutingwang@fb.com>

## Summary

- Add small changes to the onboarding documents, including [Prerequisites](https://magma.github.io/magma/docs/next/basics/prerequisites), [Quick Start Guide](https://magma.github.io/magma/docs/next/basics/quick_start_guide), and [Development Workflow](https://magma.github.io/magma/docs/next/contributing/contribute_workflow), to make the instructions more exhaustive
- Fix invalid link in **Installing Magma Locally** section of [Development Workflow](https://magma.github.io/magma/docs/next/contributing/contribute_workflow) to point to the right url
- Update all Git and shell aliases in [Development Workflow](https://magma.github.io/magma/docs/next/contributing/contribute_workflow) to only use alphanumeric symbols and dashes (only symbols allowed for Git aliases).

### CODE REVIEW

File: docs/readmes/basics/quick_start_guide.md [hunk 1]
```diff
@@ -197,7 +201,8 @@ NMS can take upto 60 seconds to finish starting up.
 
 You will probably want to enable this organization (magma-test) to access all networks,
 so go to [master.localhost](https://master.localhost) and login with the same credentials.
-Once there, you can click on the organization and then select "Enable all networks".
+Once there, you can click on the "Organizations" tab on the left sidebar, choose
+"magma-test", and then check "Enable all networks" in the subsequent pop-up window.
```
Comment (hcgatewood) [inline docs/readmes/basics/quick_start_guide.md]:
```suggestion
`magma-test`, and then check "Enable all networks" in the subsequent pop-up window.
```

File: docs/readmes/contributing/contribute_workflow.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ This opinionated workflow simplifies the complexities of dealing with Git direct
 It does this by keeping both `your_dev_branch` and `your_dev_branch_base` branches, allowing a straightforward mechanism for rebasing commit stacks.
 
 ```bash
-# Note: see below for the aliases used in this code block
+# Note: see below for the git aliases used in this code block
```
Comment (hcgatewood) [inline docs/readmes/contributing/contribute_workflow.md]:
```suggestion
# Note: see below for the Git and shell aliases used in this code block
```

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Thanks for adding! Two small changes then I'm ready to approve

**Comment by electronjoe** [comment]
Thank you for the corrective documentation @wangyyt1013 !

**Review by hcgatewood — APPROVED** [review]
Looks great!

**Pull request merged**